db/create_tables.sql

```diff
@@ -1,4 +1,4 @@
-CREATE TABLE IF NOT EXISTS `audit` (
+CREATE TABLE IF NOT EXISTS `audit_records` (
   `id` BIGINT(20) NOT NULL AUTO_INCREMENT,
   `state_machine_name` VARCHAR(255) NOT NULL,
   `state_machine_instance_id` VARCHAR(255) NOT NULL,
```

Comment:
we can have created_at.
It is the second one.
<sm_id,state_id,retryAttempt:0,state_status:Failed,..)
<sm_id,state_id,retryAttempt:1,state_status:Completed)